Tests/Weaver/Realm.FakeForWeaverTests/Realm.FakeForWeaverTests.csproj

```diff
@@ -1,38 +1,39 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
-  <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
-    <AssemblyName>Realm</AssemblyName>
-  </PropertyGroup>
+    <PropertyGroup>
```

Comment:
Why do we need these changes?

Reply:
The only difference is that I've added the "Include" to the Generated attribute. It looks like this because I run the autoformat on the file